src/components/button-toggle/README.md

```diff
@@ -0,0 +1,97 @@
+# MdButtonToggle
+
+`MdButtonToggle` is a group of buttons that can be toggled. There are two modes, `multiple` and
+`exclusive`. `Multiple` is a group in which multiple buttons can be toggled at once. `Exclusive` is
+a group in which only one button can be toggled at once.
+
+## `<md-button-toggle>`
+
+### Bound Properties
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `id` | string | The unique ID of the toggle. IDs are generated by default when not specified. |
+| `name` | string | Optional, defaults to parent group name if one exists for exclusive selection toggles, otherwise null. This is used to differentiate between different groups. |
+| `value` | any | Value of the toggle. Only used when in a group to determine which are selected. |
+| `checked` | boolean | Optional, default = `false`. Whether or not the toggle is checked. |
+| `disabled` | boolean | Optional, default = `false`. Whether or not the toggle is disabled |
+
+### Events
+
+| Name | Description |
+| --- | --- |
+| `change` | Emitted when the `checked` value is changed. |
+
+## `<md-button-toggle-group>`
+
+### Bound Properties
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `name` | string | Optional, the name of the group. |
+| `disabled` | boolean | Optional, default = `false`. |
+| `value` | any | The current value for the group. Will be set to the value of the selected toggle or a list of values from the selected toggles. |
+| `selected` | `mdButtonToggle` | The current selected toggle or a list of the selected toggles. |
+
+### Attributes
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `multiple` | boolean | Optional, default = `false`. Whether or not the group allows multiple selection. |
+
+### Events
+
+| Name | Description |
+| --- | --- |
+| `change` | Emitted when the `value` of the group changes. |
+
+## Usage
+
+### Basic Usage
+
+`md-button-toggle` can be used on its own and acts as a checkbox.
+
+```html
+<md-button-toggle>Bold</md-button-toggle>
+```
+
+### Multiple Selection
+
+`md-button-toggle` can be used in a multiple selection group when surrounded by
+`md-button-toggle-group multiple`. This styles all buttons within the group to appear as a single
+group of button toggles.
+
+```html
+<md-button-toggle-group multiple>
+    <md-button-toggle>Flour</md-button-toggle>
+    <md-button-toggle>Eggs</md-button-toggle>
+    <md-button-toggle>Sugar</md-button-toggle>
+    <md-button-toggle>Milk</md-button-toggle>
+</md-button-toggle-group>
```

Comment:
Nitpick: It might be a little confusing that you are showing a disabled button-toggle along with a selected one.  It almost looks like multi-select.  Perhaps create another section for the "disabled" example, so it's clear?